.github/workflows/web_app-ci.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name: Deploy Infrastructure and Web Application
+
+on: [push, pull_request] # yamllint disable-line rule:truthy
+
+env:
+  AWS_REGION: 'us-west-2'
+  IMAGE_NAME: 'banigoden/web_app'
+  IMAGE_VERSION: ${{ github.run_number }}
+  BRANCH_NAME: ${{ github.ref_name }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
```

Comment:
let's make first steps for buildiong and publishing docker, after terraform and anible etc.

Reply:
moved from pre-commit to deploy